### DISCUSSION TITLE
Thoughts on coming H.266 codec?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [staxrip/staxrip#449] [General]
https://www.youtube.com/watch?v=wLo4sN1DCN0

**Comment by Dendraspis** [comment]
#266

**Reply by ZeroCool22** [reply]
![872be169144077120242844098_700wa_0](https://user-images.githubusercontent.com/13344308/104744482-b2359b80-572b-11eb-8b7c-16c2a985e700.gif)

**Reply by ZeroCool22** [reply]
> 
> 
> #266

H266 supports **GPU** encoding?

**Reply by JJKylee** [reply]
In the end, yes. In the near future, definitely no.

Even AV1 hardware **encoding** is not implemented yet, and there's no detailed roadmap for it now. Only AV1 hardware **decoding** was just barely introduced last year by Nvidia on the Ampere architecture and other companies are following.

Talk about VVC? It's just a dream for now. 😝

**Comment by ghost** [comment]
hmm...
the codec is h266, and the thread is 266... Simple coincidence ?
I'm sure that if there is gonna be a x266 project, I'll be using it a lot!

**Comment by JJKylee** [comment]
I'm following [Doom9's thread on VVC](https://forum.doom9.org/showthread.php?t=174940) and my overall impression is that it's too early to implement in practice.

The reference encoder itself(VTM) is of course very hard to use, and AFAIK, Fraunhofer is developing both a decoder([VVdeC](https://github.com/fraunhoferhhi/vvdec)) and an encoder([VVenC](https://github.com/fraunhoferhhi/vvenc)), but they are still at a very early stage.
And rumor has it that [MulticoreWare is working on an open source project `x266`](https://ottverse.com/x266-vvc-multicoreware-opensource-encoder/) like `x265`, but nobody has seen its reality yet.

So IMHO, #266 will have to wait a loooong time before it's given attention.

**Comment by STENCv2** [comment]
Hey Fraunhofer has developed and published the VVenC & VVdeC have the admins considered adding it to staxrip???

**Reply by Dendraspis** [reply]
It's implemented since `v2.23.0 (2023-07-24)`  😅

**Reply by STENCv2** [reply]
oh damn sorry i never explored other encoders besides h265
sorry to disturb ya'll

**Reply by Dendraspis** [reply]
You may want to download the latest version and start with new settings.
I guess you know where to find the encoders?

**Reply by STENCv2** [reply]
Yes sir, i have the latest version of staxrip and found a good vvcdecoder and i am currently working on settings although theres not much to experiment with.
